unittests/integration/builder_tester.cpp

```diff
@@ -119,6 +119,8 @@ CUDAQ_TEST(BuilderTester, checkSimple) {
     EXPECT_EQ(counter, 1000);
   }
 
+#ifndef CUDAQ_BACKEND_TENSORNET_MPS
```

Comment:
Could you please file an issue for supporting to execute code with multi-controlled operations on the tensortnet backends? What we need to do is run the pass for multi-control decompositions, which requires disabling library mode by default for these backends. Let's discuss this next Tuesday in the design meeting.

Also, please make sure to make a note of this limitation in the documentation. What exactly is the error you would get if you try to execute a multi-controlled operation on these backends?

Reply:
I've added an issue to track this: https://github.com/NVIDIA/cuda-quantum/issues/884

The error message would look like: https://github.com/NVIDIA/cuda-quantum/blob/d743684c1eeff3949100cae82de864051018312f/runtime/nvqir/cutensornet/simulator_mps_register.cpp#L89 

The new docs also mentions this limitation: https://github.com/NVIDIA/cuda-quantum/pull/871/files#r1386815712
Note: this limitation is only for the `tensornet-mps` backend (not the `tensornet` backend); hence the `#ifndef CUDAQ_BACKEND_TENSORNET_MPS`.